README.md

```diff
@@ -202,13 +202,17 @@ repeating the mistakes of the past.
 - Write blog posts or speak at meetups and conferences.
 
 #### Collaboration over competition
-- We have a very collaborative culture. There are a number of reasons why this is preferable:
-    - [Competitive cultures tend to not bode well for creating diverse teams.](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a).
+- We have a very collaborative culture. [Competitive cultures tend to not bode well for creating diverse teams](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a), which we value highly.
 
 #### Favour experimentation over argument
 
 #### Be excellent to each other!
 
+#### Inclusion over exclusion
+- We value diversity - and a diverse culture is one that does not exclude people. Bullying,
+  [contempt](http://blog.aurynn.com/86/contempt-culture), and other actions that cause
+  people to feel excluded have no place in our team.
```

Comment:
Providing some examples, would be great. 
Also how do you prevent cross team exclusion? These are great points on developer -> developer relations, but interactions between different teams and disciplines occur as well and can also lead to the feeling of exclusion (intention or not). This goes beyond language preference or implementation method, not everyone may be on the same technical level and require a different method of communication/explanation.